src/frontends/tensorflow_common/src/op/segment_sum.cpp

```diff
@@ -45,9 +49,15 @@ OutputVector translate_segment_sum_op(const NodeContext& node) {
     auto num_indices = make_shared<v0::Squeeze>(segment_ids_shape, squeeze_axis);
     auto indices = make_shared<v4::Range>(const_zero, num_indices, const_one, indices_type);
 
-    auto emb_segment_sum = make_shared<v3::EmbeddingSegmentsSum>(data, indices, segment_ids, num_segments);
-    set_node_name(node.get_name(), emb_segment_sum);
-    return {emb_segment_sum};
+    auto emb_segment_sum_real = make_shared<v3::EmbeddingSegmentsSum>(data, indices, segment_ids, num_segments);
```

Comment:
it will not pass validation since EmbeddingSegmentsSum will receive input with `ComplexTypeMark`.
Please build locally your changes and run layer test to check